lib/ui/painting.dart

```diff
@@ -2037,6 +2037,52 @@ class Gradient extends Shader {
   }
   void _initRadial(double centerX, double centerY, double radius, Int32List colors, Float32List colorStops, int tileMode, Float64List matrix4) native 'Gradient_initRadial';
 
+  /// Creates a sweep gradient centered at `center` that starts at `startAngle`
+  /// and ends at `endAngle`.
+  ///
```

Comment:
You should add a paragraph that explains which direction an angle of 0 is, and in which direction positive angles go (clockwise, hopefully).

Reply:
Done. [It moves clockwise starting from the right-side x-axis]